src/render/dom/utils/supported-elements.ts

```diff
@@ -121,16 +121,68 @@ export const htmlElements = [
 ] as const
 export type HTMLElements = UnionStringArray<typeof htmlElements>
 
+/**
+ * We keep these listed seperately as we use the lowercase tag names as part
+ * of the runtime bundle to detect SVG components
+ */
+export const lowercaseSVGElements = [
```

Comment:
Would it make sense to extract the subset of these elements that are reused between `lowerCaseSVGElements` and `svgElements` to avoid the repetition?

Reply:
Yeah I wanted to but TS uses `svgElements` to generate tag name types and it doesn't work properly when it contains another spread array.